### ISSUE TITLE
Use same scaler to normalize the data

### ISSUE THREAD

**Issue by natebuel29** [natebuel29/ufc-event-predictor#22]
The current normalize function takes in a single dataset (whether that be train or test) and normalizes them independently. It is desired to normalize using the same scaler, which isn't how it functions now.

https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html
